gltf-utils/src/lib.rs

```diff
@@ -1,41 +1,24 @@
-#![allow(unknown_lints)]
-#![allow(cast_lossless)]
+//! This crate provides utility methods in addition to core gltf, such as
+//! accessor iterators and easy conversions between different representations of
+//! accessor items.
+
+#![warn(missing_docs)]
+#![warn(trivial_casts)]
+
 extern crate byteorder;
 extern crate gltf;
 
-use std::{fmt, marker};
+use std::fmt;
+use std::marker::PhantomData;
 use std::mem::size_of;
 
 use byteorder::{LE, ByteOrder};
 
 use gltf::accessor::{DataType, Dimensions};
 
-/// Helper trait for denormalizing integer types.
-///
-/// # Examples
-///
-/// Denormalize a single `u16`.
-///
-/// ```rust
-/// use gltf_utils::Denormalize;
-/// let x: u16 = 65535;
-/// assert_eq!(1.0, x.denormalize());
-/// ```
-///
-/// Denormalize an array of integers.
-///
-/// ```rust
-/// use gltf_utils::Denormalize;
-/// let rgb: [u8; 3] = [0, 120, 255];
-/// assert_eq!([0.0, 120.0 / 255.0, 1.0], rgb.denormalize());
-/// ```
-pub trait Denormalize {
```

Comment:
I think this should stay for now.

Reply:
Okay, this was moved back in and made use of.